src/UXClient/Utils.ts

```diff
@@ -931,6 +931,30 @@ class Utils {
         });
         return convertedData;
     }
+
+    static cullValuesOverMax (availabilityDistribution: any) {
+        const dateZero = '0000-01-01T00:00:00Z';
+        let maxDateValue = (new Date()).setFullYear((new Date()).getFullYear() + 10).valueOf(); // 10 years from now
```

Comment:
lets maybe make this 10 year piece a parameter?

Reply:
like maybe they pass in the max date value

Reply:
cool, I like it

Reply:
also in the explorer we cull values before a certain date (e.g. 01-01-0001) maybe we do that here too????